src/cli/mod.rs

```diff
@@ -353,6 +465,42 @@ impl LoadCli {
             LanguageField::Grumpkin => todo!(),
         }
     }
+
+    #[cfg(not(target_arch = "wasm32"))]
+    fn set_lurk_dirs(&self, config: &HashMap<String, String>) {
```

Comment:
This function is repeated. It can be extracted to a standalone function that doesn't call `self`, but receives all the data needed